app/lib/delegates/wa/murray.rb

```diff
@@ -47,21 +28,42 @@ def check_for_errors!(page)
   end
 
   def set_form_fields
-    form[PREFIX_FIELD_NAME] = ['Mr.', 'Ms.'].sample # correct sometimes
-    form[STATE_FIELD_NAME] = STATE_WA_VAL
-    form[TOPIC_FIELD_NAME] = TOPIC_ELECTIONS_VAL
-    form[SUBJECT_FIELD_NAME] = 'Democratic party superdelegate'
-
-    FIELD_NAMES.each do |k, v|
-      form[v] = message.send(k)
+    form_values.each do |label_text, value|
+      name = name_for(label_text)
+      form[name] = value
     end
   end
 
+  def name_for(label_text)
+    page
+      .labels
+      .detect { |l| l.text == label_text }
+      .node['for']
+  end
+
+  def form_values
+    {
+      'Prefix *' => ['Mr.', 'Ms.'].sample, # correct sometimes,
```

Comment:
@philipbjorge had a question about adding this to the data model instead of randomly assigning gender (which probably would annoy me). What if we supported just a few common answers (Mr/Mrs?) and recorded that data for future forms? Or not worth it in your opinion?